sdk/lib/crt/startup/atonexit.c

```diff
@@ -20,8 +20,8 @@
 
   void __cdecl _lock (int _File);
   void __cdecl _unlock (int _File);
-  
-_PVFV *__onexitbegin;
+
+_PVFV *__onexitbegin = (_PVFV *)(LONG_PTR)-1;
```

Comment:
Looking at how `__onexitbegin` is used in the code, I noticed two "special" values it also uses: "-1" and "NULL". What is the actual difference in meaning (in the context of `__onexitbegin`˙) of these values?

Reply:
42

Reply:
For fucking sake @tkreuzer I'm asking for an actual explanation, not for jokes. It's not the first time you answer such shit in PRs,